### PR TITLE
adding fmf metadata to run test via tmt

### PR DESCRIPTION
**Pull request by odubaj**

### CODE REVIEW

File: Sanity/mariadb-odbc/main.fmf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- python3-pyodbc
+- unixODBC
+- unixODBC-libs
+- mariadb-connector-odbc
+- mariadb-server
+duration: 5m
+extra-summary: 'TestVersion:     2.0'
```
Comment (phracek) [inline Sanity/mariadb-odbc/main.fmf:18]: Just nit pick. Why so many white spaces?

File: Sanity/mariadb-odbc/main.fmf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- Filip Janus <fjanus@redhat.com>
+component:
+- python3-pyodbc
+- unixODBC
+- mariadb-connector-odbc
+- mariadb
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- python3-pyodbc
```
Comment (phracek) [inline Sanity/mariadb-odbc/main.fmf:12]: This package duplication is intended? The package is mentioned in `component` field.

File: Sanity/postgresql-odbc/main.fmf [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Honza Horak <hhorak@redhat.com>
+component:
+- python3-pyodbc
+- unixODBC
+- postgresql-odbc
+- postgresql
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- python3-pyodbc
```
Comment (phracek) [inline Sanity/postgresql-odbc/main.fmf:12]: The same duplication.

File: Sanity/mariadb-odbc/main.fmf (hunk 1 above)
Comment (odubaj) [reply Sanity/mariadb-odbc/main.fmf:18]: result of tmt test convert, didn't fixed it manually, as it is a string and might have a purpose (I do not think that either...)

File: Sanity/mariadb-odbc/main.fmf (hunk 2 above)
Comment (odubaj) [reply Sanity/mariadb-odbc/main.fmf:12]: What duplication do you mean?

File: Sanity/postgresql-odbc/main.fmf (hunk 3 above)
Comment (odubaj) [reply Sanity/postgresql-odbc/main.fmf:12]: What duplication do you mean?

File: Sanity/mariadb-odbc/main.fmf (hunk 2 above)
Comment (odubaj) [reply Sanity/mariadb-odbc/main.fmf:12]: component means for which component is the test suited to be ran

recommend means the same as requires in rpm spec -> which packages are required to be installed for the test to run properly

File: Sanity/postgresql-odbc/main.fmf (hunk 3 above)
Comment (hhorak) [reply Sanity/postgresql-odbc/main.fmf:12]: I think this is expected to have it listed twice in two different sections.

Comment (fila43) [reply Sanity/postgresql-odbc/main.fmf:12]: It could be a good idea for improvement, but for now, it's necessary, I am right?

Comment (odubaj) [reply Sanity/postgresql-odbc/main.fmf:12]: agree

**Pull request merged**